src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor.cs

```diff
@@ -34,6 +61,29 @@ public partial class BitNav
 
         protected override string RootElementClass => "bit-nav";
 
+
+        protected override async Task OnInitializedAsync()
+        {
+
+            navigationManager.LocationChanged += OnLocationChanged;
+
+
+            await base.OnInitializedAsync();
+        }
+
+        private void OnLocationChanged(object sender, LocationChangedEventArgs args)
+        {
+            var currentPage = navigationManager.Uri.Replace(navigationManager.BaseUri, string.Empty);
+
+            string currentPageKey = NavLinkItems.Where(nli => nli.Links != null).SelectMany(nli => nli.Links).FirstOrDefault(l => l.Url.ToLower().Contains(currentPage.ToLower()))?.Key ?? string.Empty;
```

Comment:
please format this line into multiple lines

Reply:
formatted into multiple lines